finance-api/overview.md

```diff
@@ -0,0 +1,18 @@
+Finance API overview
+===
+Merchants who take payments with Zettle have an Zettle account that processes transactions for the payments. For example, card payments made with Zettle card readers.
+
+The Finance API is used to fetch finance-related information that is processed through Zettle. The information includes account balance, all the transactions processed through Zettle, and payout.
+
+# Understand how card payments work at Zettle
```

Comment:
Not very important, but what about dropping the `card` here and just saying how a `payment` works?
The flow describes card payments, card refunds and APMs. All of them work the same way. Maybe integrators wouldn't be in doubt about APMs and refunds work.